client-app/builder-preview/BuilderPreview.ts

```diff
@@ -1,22 +1,22 @@
 import { App } from "vue";
-import { useTemplate } from "./pages/useTemplate";
+import { useStaticPage } from "@/core/composables";
 
 export default {
   install: (app: App, options: any) => {
     console.log("install buidler preview plugin");
     console.log(app);
-    options.router.push("/demo-page");
+    options.router.push("/");
```

Comment:
no, it's matter. because site load through /builder-preview.html. then site redirects to root. in the future it will be redirect to correct page, but now it should be so